drivers/platform/stm32/stm32_spi.c

```diff
@@ -186,7 +186,6 @@ int32_t stm32_spi_init(struct no_os_spi_desc **desc,
 	spi_desc->extra = sdesc;
 	sinit = param->extra;
 
-	csip_extra.port = sinit->chip_select_port;
```

Comment:
Also this change does not belong here but to another commit.

Reply:
Split last commit.